### PR TITLE
Blog app views

### PR DESCRIPTION
**Pull request by Estherstarr**
In this milestone, I implemented 
- [ ] Create view routes

### CODE REVIEW

File: app/views/posts/index.html.erb [hunk 1]
```diff
@@ -12,15 +12,27 @@
...
-        <h4>comment 1</h4>
-        <h4>comment 2</h4>
+  <% if @posts.blank?%>
+      <p>There are no posts</p>
+      <% else %>
+      <div class = "pb-5">
+      <h2 class = "text-2xl"> <%= @user.name %> </h2>
+        Number of posts: <%= @user.posts_counter%>
+      </div>
+          <% @posts.each do |post| %>
```
Comment (rustam-tolipov) [inline app/views/posts/index.html.erb:22]: - [ ] We kindly request that you utilize the pre-existing method, recent_comments, for displaying comments on posts. To ensure the best user experience, we ask that the comments be displayed in chronological order. Thank you for your attention to this matter. 😎

File: app/views/users/show.html.erb [hunk 2]
```diff
@@ -4,24 +4,31 @@
...
-
-
+  <h1 class = "pb-5 text-2xl">USERS</h1>
+      <div class = "pb-5">
+      <h2 class = "text-xl"> <%= @user.name %> </h2>
+        Number of posts: <%= @user.posts_counter%>
+      </div>
+    <h1>Bio </h1>
+    <p class = "pb-5"><%= @user.bio %></p>
+     <% @user.posts.each do |post| %>
```
Comment (rustam-tolipov) [inline app/views/users/show.html.erb]: - [ ] We would like to bring to your attention that the user/show page requires an update to display recent posts. Kindly utilize the appropriate methods to achieve this ( _I think you have created such method for it_ ). The comments should be displayed in chronological order to provide the best user experience. Thank you for your attention to this matter. 😊

**Review by rustam-tolipov — CHANGES_REQUESTED** [review]
Hello @Estherstarr,

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
_Your project is almost complete!_
_No Linters errors_

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Check the comments under the review._

You can also consider:
N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by EricMbouwe — APPROVED** [review]
Hi @Estherstarr, 

Your project is complete! ⚡🔥  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights 🎉🔥
- Requested changes ✔
- No linter errors ✔

## Optional suggestions ☂

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**